include/Contact.php

```diff
@@ -85,7 +85,7 @@ function contact_remove($id) {
 
 function terminate_friendship($user,$self,$contact) {
```

Comment:
Standards: Could you please add a space after each comma in the parameter list? ☺️